Engine/src/Engine/ECS/System.h

```diff
@@ -0,0 +1,25 @@
+/*****************************************************************//**
+\file       System.h
+\project    INSERT_PROJECT_HERE
+\author     Lim Guan Hui, l.guanhui , 2000552
+\par        email: l.guanhui@digipen.edu
+\date       2/7/2021
+\brief  
+Base class to be inherited from when creating new systems 
+
+Copyright (C) 2021 DigiPen Institute of Technology.
+Reproduction or disclosure of this file or its contents
+without the prior written consent of DigiPen Institute of
+Technology is prohibited.
+ *********************************************************************/
+#pragma once
+#include <set>
+#include "ECS_Utility.h"
+class System
```

Comment:
Should Systems be wrapped in engine namespace or not?

Reply:
It has been wrapped